Makefile

```
@@ -4,7 +4,7 @@
 # - use the VERSION as arg of the bundle target (e.g make bundle VERSION=0.0.2)
 # - use environment variables to overwrite this value (e.g export VERSION=0.0.2)
 
-_VERSION := $(shell git describe --tags --dirty)
+_VERSION := $(shell git describe --tags --dirty --always)
```

I like the idea of syncing the version to the tags.

But if we stick to "release tags" for "accepted" versions I would instead do:

```suggestion
_VERSION := $(shell git describe --tags)
```

as there ShouldNot™ be an image without its mathing repo tag - but if there is, you would still get the long format of `git describe` with num commits and hash